### DISCUSSION TITLE
Alert Catalogue of Fastlane errors

### DISCUSSION THREAD

**Discussion by ykhandelwal913** [fastlane/fastlane#22266] [Ideas]
I am not sure if Fastlane already has a alert catalogue where they specify the fastlane errors and possible cause of the errors which can help identify the issue in build cycle. 
If the catalogue is not there, is there any plan to have one?

**Comment by sonia598lewis** [comment]
Hello,
Fastlane does not currently have a dedicated alert  [MedicareCard](https://www.imedicarecard.com) catalog for errors and their causes. However, there is an ongoing discussion on GitHub about creating one. You might want to contribute to or follow that discussion for updates.

Best Regards,
Sonia Lewis

**Comment by ykhandelwal913** [comment]
@sonia598lewis can you share the discussion thread..
